 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container rinha-postgres  Creating
 Container rinha-postgres  Created
 Container cleverton-rocha-java-api02-1  Creating
 Container cleverton-rocha-java-api01-1  Creating
 Container cleverton-rocha-java-api02-1  Created
 Container cleverton-rocha-java-api01-1  Created
 Container cleverton-rocha-java-nginx-1  Creating
 Container cleverton-rocha-java-nginx-1  Created
Attaching to api01-1, api02-1, nginx-1, rinha-postgres
rinha-postgres  | The files belonging to this database system will be owned by user "postgres".
rinha-postgres  | This user must also own the server process.
rinha-postgres  | 
rinha-postgres  | The database cluster will be initialized with locale "en_US.utf8".
rinha-postgres  | The default database encoding has accordingly been set to "UTF8".
rinha-postgres  | The default text search configuration will be set to "english".
rinha-postgres  | 
rinha-postgres  | Data page checksums are disabled.
rinha-postgres  | 
rinha-postgres  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
rinha-postgres  | creating subdirectories ... ok
rinha-postgres  | selecting dynamic shared memory implementation ... posix
rinha-postgres  | selecting default max_connections ... 100
rinha-postgres  | selecting default shared_buffers ... 128MB
rinha-postgres  | selecting default time zone ... Etc/UTC
rinha-postgres  | creating configuration files ... ok
rinha-postgres  | running bootstrap script ... ok
rinha-postgres  | performing post-bootstrap initialization ... ok
nginx-1         | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1         | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1         | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1         | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1         | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1         | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1         | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1         | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1         | /docker-entrypoint.sh: Configuration complete; ready for start up
rinha-postgres  | syncing data to disk ... ok
rinha-postgres  | 
rinha-postgres  | 
rinha-postgres  | Success. You can now start the database server using:
rinha-postgres  | 
rinha-postgres  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
rinha-postgres  | 
rinha-postgres  | initdb: warning: enabling "trust" authentication for local connections
rinha-postgres  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
rinha-postgres  | waiting for server to start....2024-02-26 20:39:29.586 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
rinha-postgres  | 2024-02-26 20:39:29.586 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-postgres  | 2024-02-26 20:39:29.588 UTC [51] LOG:  database system was shut down at 2024-02-26 20:39:29 UTC
rinha-postgres  | 2024-02-26 20:39:29.591 UTC [48] LOG:  database system is ready to accept connections
rinha-postgres  |  done
rinha-postgres  | server started
rinha-postgres  | CREATE DATABASE
rinha-postgres  | 
rinha-postgres  | 
rinha-postgres  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
rinha-postgres  | CREATE EXTENSION
rinha-postgres  | CREATE TABLE
rinha-postgres  | CREATE TABLE
rinha-postgres  | INSERT 0 5
rinha-postgres  |  pg_prewarm 
rinha-postgres  | ------------
rinha-postgres  |           1
rinha-postgres  | (1 row)
rinha-postgres  | 
rinha-postgres  |  pg_prewarm 
rinha-postgres  | ------------
rinha-postgres  |           0
rinha-postgres  | (1 row)
rinha-postgres  | 
rinha-postgres  | 
rinha-postgres  | 
rinha-postgres  | waiting for server to shut down....2024-02-26 20:39:29.843 UTC [48] LOG:  received fast shutdown request
rinha-postgres  | 2024-02-26 20:39:29.843 UTC [48] LOG:  aborting any active transactions
rinha-postgres  | 2024-02-26 20:39:29.844 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
rinha-postgres  | 2024-02-26 20:39:29.845 UTC [49] LOG:  shutting down
rinha-postgres  | 2024-02-26 20:39:29.845 UTC [49] LOG:  checkpoint starting: shutdown immediate
rinha-postgres  | 2024-02-26 20:39:29.850 UTC [49] LOG:  checkpoint complete: wrote 939 buffers (7.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.005 s, sync=0.001 s, total=0.006 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4301 kB, estimate=4301 kB; lsn=0/191DE78, redo lsn=0/191DE78
rinha-postgres  | 2024-02-26 20:39:29.854 UTC [48] LOG:  database system is shut down
rinha-postgres  |  done
rinha-postgres  | server stopped
rinha-postgres  | 
rinha-postgres  | PostgreSQL init process complete; ready for start up.
rinha-postgres  | 
rinha-postgres  | 2024-02-26 20:39:29.960 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
rinha-postgres  | 2024-02-26 20:39:29.960 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
rinha-postgres  | 2024-02-26 20:39:29.960 UTC [1] LOG:  listening on IPv6 address "::", port 5432
rinha-postgres  | 2024-02-26 20:39:29.960 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-postgres  | 2024-02-26 20:39:29.962 UTC [66] LOG:  database system was shut down at 2024-02-26 20:39:29 UTC
rinha-postgres  | 2024-02-26 20:39:29.964 UTC [1] LOG:  database system is ready to accept connections
nginx-1         | 2024/02/26 20:39:32 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:32 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:34 [error] 28#28: *4 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 
api01-1         |   .   ____          _            __ _ _
api01-1         |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api01-1         | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api01-1         |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api01-1         |   '  |____| .__|_| |_|_| |_\__, | / / / /
api01-1         |  =========|_|==============|___/=/_/_/_/
api01-1         |  :: Spring Boot ::                (v3.2.3)
api01-1         | 
api02-1         | 
api02-1         |   .   ____          _            __ _ _
api02-1         |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api02-1         | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api02-1         |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api02-1         |   '  |____| .__|_| |_|_| |_\__, | / / / /
api02-1         |  =========|_|==============|___/=/_/_/_/
api02-1         |  :: Spring Boot ::                (v3.2.3)
api02-1         | 
api01-1         | 2024-02-26T20:39:34.833Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : Starting Rinhadebackend2024q1Application v0.0.1 using Java 17.0.2 with PID 1 (/usr/src/app/rinhadebackend2024q1-0.0.1.jar started by root in /usr/src/app)
api01-1         | 2024-02-26T20:39:34.917Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : No active profile set, falling back to 1 default profile: "default"
api02-1         | 2024-02-26T20:39:35.131Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : Starting Rinhadebackend2024q1Application v0.0.1 using Java 17.0.2 with PID 1 (/usr/src/app/rinhadebackend2024q1-0.0.1.jar started by root in /usr/src/app)
api02-1         | 2024-02-26T20:39:35.217Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : No active profile set, falling back to 1 default profile: "default"
nginx-1         | 2024/02/26 20:39:36 [error] 28#28: *5 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:38 [error] 28#28: *6 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:39:40.021Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
nginx-1         | 2024/02/26 20:39:40 [error] 28#28: *7 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1         | 2024-02-26T20:39:40.222Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
api01-1         | 2024-02-26T20:39:40.328Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 296 ms. Found 2 JPA repository interfaces.
api02-1         | 2024-02-26T20:39:40.616Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 387 ms. Found 2 JPA repository interfaces.
nginx-1         | 2024/02/26 20:39:42 [error] 28#28: *8 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:39:43.717Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
api01-1         | 2024-02-26T20:39:43.734Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api01-1         | 2024-02-26T20:39:43.734Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.19]
api02-1         | 2024-02-26T20:39:44.118Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
nginx-1         | 2024/02/26 20:39:44 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:44 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.4:8080/clientes/1/extrato", host: "localhost:9999"
api02-1         | 2024-02-26T20:39:44.134Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api02-1         | 2024-02-26T20:39:44.134Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.19]
api01-1         | 2024-02-26T20:39:44.429Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api01-1         | 2024-02-26T20:39:44.430Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 9112 ms
api02-1         | 2024-02-26T20:39:45.022Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api02-1         | 2024-02-26T20:39:45.023Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 9396 ms
nginx-1         | 2024/02/26 20:39:46 [error] 28#28: *12 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:39:46.222Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api01-1         | 2024-02-26T20:39:46.523Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.4.Final
api02-1         | 2024-02-26T20:39:46.633Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api01-1         | 2024-02-26T20:39:46.721Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
api02-1         | 2024-02-26T20:39:46.840Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.4.Final
api02-1         | 2024-02-26T20:39:47.026Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
api01-1         | 2024-02-26T20:39:47.723Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api02-1         | 2024-02-26T20:39:47.927Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api01-1         | 2024-02-26T20:39:47.931Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
api02-1         | 2024-02-26T20:39:48.125Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
nginx-1         | 2024/02/26 20:39:48 [error] 28#28: *13 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:39:48.926Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@fcd0e8d
api01-1         | 2024-02-26T20:39:48.927Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api01-1         | 2024-02-26T20:39:49.017Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
api02-1         | 2024-02-26T20:39:49.233Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@fcd0e8d
api02-1         | 2024-02-26T20:39:49.234Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api02-1         | 2024-02-26T20:39:49.336Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
nginx-1         | 2024/02/26 20:39:50 [error] 28#28: *14 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:52 [error] 28#28: *15 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:54 [error] 28#28: *16 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:39:54.422Z  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
api01-1         | 2024-02-26T20:39:54.622Z  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
api02-1         | 2024-02-26T20:39:55.927Z  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
nginx-1         | 2024/02/26 20:39:56 [error] 28#28: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:39:56 [error] 28#28: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.4:8080/clientes/1/extrato", host: "localhost:9999"
api02-1         | 2024-02-26T20:39:56.217Z  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
api01-1         | 2024-02-26T20:39:57.119Z  WARN 1 --- [           main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
nginx-1         | 2024/02/26 20:39:58 [error] 28#28: *20 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1         | 2024-02-26T20:39:58.624Z  WARN 1 --- [           main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
api01-1         | 2024-02-26T20:39:59.328Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port 8080 (http) with context path ''
api01-1         | 2024-02-26T20:39:59.339Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : Started Rinhadebackend2024q1Application in 27.309 seconds (process running for 30.648)
nginx-1         | 2024/02/26 20:40:00 [error] 28#28: *21 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1         | 2024-02-26T20:40:00.529Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port 8080 (http) with context path ''
api02-1         | 2024-02-26T20:40:00.618Z  INFO 1 --- [           main] c.r.r.Rinhadebackend2024q1Application    : Started Rinhadebackend2024q1Application in 28.682 seconds (process running for 31.995)
api02-1 exited with code 137
nginx-1         | 2024/02/26 20:40:02 [error] 28#28: *22 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:40:04 [error] 28#28: *23 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:40:06 [error] 28#28: *24 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1         | 2024/02/26 20:40:26 [error] 28#28: *25 connect() failed (113: No route to host) while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.3:8080/clientes/1/extrato", host: "localhost:9999"
api01-1         | 2024-02-26T20:40:26.932Z  INFO 1 --- [nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
api01-1         | 2024-02-26T20:40:26.933Z  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
api01-1         | 2024-02-26T20:40:26.934Z  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
nginx-1         | 2024/02/26 20:40:27 [error] 28#28: *25 upstream prematurely closed connection while reading response header from upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.24.0.4:8080/clientes/1/extrato", host: "localhost:9999"
rinha-postgres  | 2024-02-26 20:40:27.737 UTC [70] LOG:  unexpected EOF on client connection with an open transaction
api01-1 exited with code 137
nginx-1         | 2024/02/26 20:40:29 [error] 28#28: *28 no live upstreams while connecting to upstream, client: 172.24.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
rinha-postgres  | 2024-02-26 20:40:32.390 UTC [1] LOG:  received fast shutdown request
rinha-postgres  | 2024-02-26 20:40:32.390 UTC [1] LOG:  aborting any active transactions
rinha-postgres  | 2024-02-26 20:40:32.392 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
rinha-postgres  | 2024-02-26 20:40:32.392 UTC [64] LOG:  shutting down
rinha-postgres  | 2024-02-26 20:40:32.392 UTC [64] LOG:  checkpoint starting: shutdown immediate
rinha-postgres  | 2024-02-26 20:40:32.393 UTC [64] LOG:  checkpoint complete: wrote 93 buffers (0.8%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.002 s, sync=0.001 s, total=0.002 s; sync files=0, longest=0.000 s, average=0.000 s; distance=144 kB, estimate=144 kB; lsn=0/1941EA0, redo lsn=0/1941EA0
rinha-postgres  | 2024-02-26 20:40:32.397 UTC [1] LOG:  database system is shut down
rinha-postgres exited with code 0
